frontend/packages/data-portal/app/components/Filters/MultiInputFilter.tsx

```diff
@@ -74,7 +76,9 @@ export function MultiInputFilter({
             const value = values[filter.id]
 
             if (value) {
-              prev.set(filter.queryParam, value)
+              // Our filters currently support numeric IDs and use the queryParam as the key
+              // The filter will show the prefix, but we do not need to store it in the query params
+              prev.set(filter.queryParam, extractNumericId(value) ?? '')
```

Comment:
Does this affect the author filter? Little concerned cuz the component itself sounds more like a generic filter thing but now it seems to do some pretty ID specific stuff. I wonder if we should add a property to `FilterInputData` for a regex or something? similar to what @kne42 did with `<EntityIdFilter>`?

Reply:
@bchu1 I just changed it to use a `removeIdPrefix` function instead which first checks to see if the queryParam has a prefix and if it doesn't, just return the input.  Are there instances where we'll have prefix-someString instead of prefix-number?  That won't work with this approach.

I could add a prop - it's not my favorite because I find it hard to click through the editor to find where the props get set.  More flexible though.

Thoughts?

Reply:
I'm ok with the regex (maybe add a comment on the top of the component?), just wondering if this breaks the author name filter cuz that's just a string. Or is that a different component?

Reply:
It previously did break the author name, but it no longer breaks it because it skips any fields that don't have prefixes defined.